azure-pipelines.yml

```diff
@@ -39,6 +39,8 @@ jobs:
     displayName: 'Install dependencies'
 
   - script: |
+      mkdir temp
```

Comment:
If we plan to keep this, do you think you can add a comment explaining why this is here?

Because if I did not know the context I would believe this to be "dead code" that was not removed.

Reply:
Yes you're right!